src/lint/features.md

```diff
@@ -10,6 +10,36 @@ Below are listed the currently supported rules for linting:
 - **NSP.unary**: no spaces between a unary operator and its operand
 - **NSP.TrailingWhitespace**: no spaces between the last token in a line and the corresponding newline `\n`
 
+## Indentation
+- **IN1**: Lines are indented a fixed amount of spaces for each indentation level. Defaults to 4, can be set to a custom value
```

Comment:
Probably correct to use the descriptive names here, with the "identifier" (or whatever we want to call 'IN1') is also mentioned (good because it ties the doc to the source file)